test/apiv2/20-containers.at

```diff
@@ -98,9 +98,41 @@ t GET libpod/images/newrepo:v2/json 200 \
   .Comment=bar \
   .Config.Cmd[-1]="/bin/foo"
 
+# Create a container for testing the container initializing later
+podman create -t -i --name myctr alpine ls
```

Comment:
I'd prefer to see `$IMAGE` here, not alpine. `$IMAGE` is already predefined as an image that will be prefetched and present; `alpine` will require pulling from docker.io, which is a major failure point.